tutel/custom/custom_kernel.cpp

```diff
@@ -321,6 +329,14 @@ static void init_nccl(
   g_cuda_events.resize(num_split);
   g_num_slices_per_split = num_slices_per_split;
   g_world_size = world_size;
+  g_world_rank = world_rank;
+
+  if (const char* local_size = std::getenv("LOCAL_SIZE")) {
```

Comment:
When is this env set?

Reply:
when launch with `tutel.launcher.run` https://github.com/microsoft/tutel/blob/befca8be7bf45a3a33fa4e2091b043220209a785/tutel/launcher/run.py#L9

if using `torch.distributed.run`, need to set manually when not using all gpus on the node